lib/fluent/time.rb

```diff
@@ -111,7 +111,8 @@ def self.eq?(a, b)
     end
 
     def self.now
-      from_time(Time.now)
+      now = Fluent::Clock.real_now(:nanosecond)
```

Comment:
self.now is called frequently so how about call `Process.clock_gettime` directly to reduce function call?

Reply:
fixed https://github.com/fluent/fluentd/pull/2557/commits/809af50a26d8477d5081908cd604694667734111